runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -287,6 +287,17 @@ default boolean swapHarpoon()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "swapBait",
+		name = "Bait",
+		description = "Swap Small Net, Net, Lure with with Bait on Fishing spot",

```

Comment:
I'll commot this with GitHub's UI
```suggestion
		description = "Swap Small Net, Net, Lure with Bait on Fishing spot",
```